subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -370,490 +370,64 @@ Once a configuration is resolved it is immutable. Changing its state or the stat
 
-++++
-<sample id="defineRepository" dir="userguide/artifacts/defineRepository" title="Accessing a repository">
-                <sourcefile file="build.gradle" snippet="lookup-resolver"/>
-            </sample>
-++++
+This example demonstrates how to declare various shortcut and custom URL repositories for a project:
 
-To configure a repository:
-
-++++
-<sample id="defineRepository" dir="userguide/artifacts/defineRepository" title="Configuration of a repository">
-                <sourcefile file="build.gradle" snippet="configure-resolver"/>
-            </sample>
 ++++
-
-
-[[sub:more_about_ivy_resolvers]]
-==== More about Ivy resolvers
-
-Gradle is extremely flexible regarding repositories:
-
-* There are many options for the protocol to communicate with the repository (e.g. filesystem, http, ssh, sftp ...)
-* The protocol sftp currently only supports username/password-based authentication.
-* Each repository can have its own layout.
-
-Let's say, you declare a dependency on the `junit:junit:3.8.2` library. Now how does Gradle find it in the repositories? Somehow the dependency information has to be mapped to a path. In contrast to Maven, where this path is fixed, with Gradle you can define a pattern that defines what the path will look like. Here are some examples:footnote:[At http://ant.apache.org/ivy/history/latest-milestone/concept.html[] you can learn more about ivy patterns.]
-
-[source]
-----
-
-// Maven2 layout (if a repository is marked as Maven2 compatible, the organization (group) is split into subfolders according to the dots.)
-someroot/[organisation]/[module]/[revision]/[module]-[revision].[ext]
-
-// Typical layout for an Ivy repository (the organization is not split into subfolder)
-someroot/[organisation]/[module]/[revision]/[type]s/[artifact].[ext]
-
-// Simple layout (the organization is not used, no nested folders.)
-someroot/[artifact]-[revision].[ext]
-
-----
-
-
-To add any kind of repository (you can pretty easy write your own ones) you can do:
-
-++++
-<sample id="fileSystemResolver" dir="userguide/artifacts/excludesAndClassifiers" title="Definition of a custom repository">
-                <sourcefile file="build.gradle" snippet="file-system-resolver"/>
-            </sample>
+<sample id="multiple-repositories" dir="userguide/repositories/declaringMultipleRepositories" title="Declaring multiple repositories">
+    <sourcefile file="build.gradle" snippet="multiple-repositories"/>
+</sample>
 ++++
 
-An overview of which Resolvers are offered by Ivy and thus also by Gradle can be found http://ant.apache.org/ivy/history/latest-milestone/settings/resolvers.html[here]. With Gradle you just don't configure them via XML but directly via their API.
+The order of declaration determines how Gradle will check for dependencies at runtime. You can learn more about <<sec:dependency_resolution,Gradle's resolution mechanism>> below.
```

Comment:
This is a really important point, so I think we should highlight it somehow, perhaps with an admonition.

Reply:
:o: Avoid positional descriptions like "below"

Reply:
I knew that you'd complain about "below" but I couldn't come up with a good alternative. Leaving out "below" makes the sentence sound kind of weird. Generally speaking, I don't mind positional description. Yes, it might need to change if things are shuffled around but from my perspective that's the job of the person changing the text + the review to ensure that those positional descriptions still work.

Reply:
RE: admonition - done. BTW: Do we support other admonitions than `NOTE`? Thinking about `WARNING`, `IMPORTANT`, `TIP` etc.

Reply:
There are

> note, caution, warning, tip, and important

according to http://www.sagehill.net/docbookxsl/CustomAdmonitions.html